CMakeLists.txt

```diff
@@ -1949,6 +1956,9 @@ target_link_libraries(mixxx-lib PRIVATE ReplayGain)
 
 # Reverb
 add_library(Reverb STATIC EXCLUDE_FROM_ALL lib/reverb/Reverb.cc)
+if(MSVC)
+  target_compile_definitions(Reverb PUBLIC _USE_MATH_DEFINES)
```

Comment:
Do we need this flags just for these libraries? If so, can we make these PRIVATE ?

Reply:
I will try to restrict or remove some of them. Trial and error.